### DISCUSSION TITLE
Add team avatar using url

### DISCUSSION THREAD

**Discussion by anuj9196** [grafana/grafana#44028] [General]
Using **Docker v8.3.3 (30bb7a93ca)**

While creating a team, there is an email field that is used to get Gravatar and use as a Team avatar.
This requires the users to have a separate email to be used for Gravatar only.

Instead, support for Avatar upload, or use HTTP URL as an avatar for the team can be provided.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
